docs/catalog/cloud/evaluate-whether-to-use-TLS-termination.md

```diff
@@ -0,0 +1,39 @@
+---
+version: 1.0
+submitted_by: yelghali
+published_date: tbd
+category: cloud
+tags: 
+- security
+- compute
+- kubernetes
+- role:software-engineer
+- size:medium
+---
+
+# Evaluate whether to use TLS termination
```

Comment:
Consider changing to something like `Terminate TLS at border gateway`. We look for titles to be action-oriented on a resource and can move the discussion of whether or not you should (or can) to the assumptions and considerations sections.